sql_generators/kpi_support_metrics/templates/active_users.view.sql

```diff
@@ -0,0 +1,53 @@
+{{ header }}
+CREATE OR REPLACE VIEW
+  `{{ project_id }}.{{ dataset }}.{{ name }}`
+AS
+SELECT
+  *,
+  CASE
+    WHEN LOWER(isp) = 'browserstack'
+      THEN CONCAT("{{ friendly_name }}", ' ', isp)
+    ELSE "{{ friendly_name }}"
+  END AS app_name,
```

Comment:
This needs to account for the Mozilla Online case, too. See, for example, [the desktop query](https://github.com/mozilla/bigquery-etl/blob/4ebf436f057ffb5d4a9d73c4e56a217c19469b6a/sql/moz-fx-data-shared-prod/telemetry/clients_last_seen_v2/view.sql#L30).

```suggestion
  CASE
    WHEN LOWER(isp) = 'browserstack'
      THEN CONCAT("{{ friendly_name }}", ' ', isp)
    WHEN LOWER(distribution_id) = 'mozillaonline'
      THEN CONCAT("{{ friendly_name }}", ' ', distribution_id)
    ELSE "{{ friendly_name }}"
  END AS app_name,
```

Reply:
ah right, I must have copied it over from the firefox_ios version. Added now.